assets/scripts/app/PrintContainer.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+import { attachPrintEventListeners } from './print'
+import { getStreetImage } from '../streets/image'
+
+class PrintContainer extends React.PureComponent {
+  constructor (props) {
+    super(props)
+
+    this.createPrintImage = this.createPrintImage.bind(this)
+  }
+
+  componentDidMount () {
+    attachPrintEventListeners()
```

Comment:
What do you think about just moving the code that attaches the listeners here? Also, maybe we can use arrow functions for handlers like createPrintImage? https://medium.com/@machnicki/handle-events-in-react-with-arrow-functions-ede88184bbb